go/client/wallet_api_handler.go

```diff
@@ -512,3 +570,66 @@ func (c *batchOptions) Check() error {
 
 	return nil
 }
+
+type sendPathPaymentOptions struct {
+	Recipient        string `json:"recipient"`
+	Amount           string `json:"amount"`
+	SourceAsset      string `json:"source-asset"`
+	DestinationAsset string `json:"destination-asset"`
+	FromAccountID    string `json:"from-account-id"`
+	Message          string `json:"message"`
+	MemoText         string `json:"memo-text"`
+}
+
+func (c *sendPathPaymentOptions) Check() error {
+	// Note: we don't validate assets in here, since we need the parsed asset values in `sendPathPayment`
```

Comment:
Not sure if this is the best way to validate all input. Would love to get a second opinion on this.

Reply:
LGTM